hack/start_flotta.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+start_operator() {
+  read -p "Enter operator path: " OPERATOR_PATH
+  cd $OPERATOR_PATH
+  read -p "Enter image registry and tag: (for example IMG=quay.io/jdzon/flotta-operator:latest) " IMG
+  IMG=$IMG make docker-build docker-push
+  if [[ "$TARGET" -eq "1" ]]; then
+      IMG=$IMG TARGET=k8s make deploy
+  elif [[ "$TARGET" -eq "2" ]]; then
+      IMG=$IMG TARGET=ocp make deploy
+  fi
+  make get-certs
+}
+
+start_agent() {
+  read -p "Enter device-worker path: " DEVICE_WORKER_PATH
+  cd $DEVICE_WORKER_PATH
+  make uninstall
+  make install
+
+  read -p "Enter yggdrasil path: " YGGDRASIL_PATH
+  cd $YGGDRASIL_PATH
+  git checkout cdde836c519ac72302714b5f7ae90e45de7c79a7
+  export PREFIX=/var/local
+  make bin
+  sudo chown root.root /tmp/*.pem
+
+  if [[ "$TARGET" -eq "1" ]]; then
+    sudo ./yggd \
+      --log-level info \
+      --protocol http \
+      --path-prefix api/flotta-management/v1 \
+      --client-id $(cat /etc/machine-id) \
+      --cert-file /tmp/cert.pem \
+      --key-file /tmp/key.pem \
+      --server flotta-operator.srv
+
+  elif [[ "$TARGET" -eq "2" ]]; then
+    HTTP_SERVER=$(oc get routes flotta-operator-controller-manager -n flotta --no-headers -o=custom-columns=HOST:.spec.host)
+    sudo ./yggd \
+      --log-level info \
+      --protocol http \
+      --path-prefix api/flotta-management/v1 \
+      --client-id $(cat /etc/machine-id) \
+      --cert-file /tmp/cert.pem \
+      --key-file /tmp/key.pem \
+      --server $HOST_SERVER
+
+  fi
+}
+
+start_all() {
+  start_operator
+  start_agent
+}
+
+read -p "Choose Flotta component to start: (1 - operator, 2 - agent, 3 - all) " OP
```

Comment:
tl;dr; Please make it non-interactive. 

During dev I often tear down the env and set it up again; answering same questions every time will be very annoying and counter-productive. 

The flow I would like to follow:
 1. set-up env by providing all the input values in one command;
 2. tear down the dev
 3. set-up env by navigating command history and using command I used before (maybe changing one parameter)
 4. goto 2

Reply:
So with this approach, I think it will be better to split the script since the arguments are different for each purpose